specs/fault-proof/stage-one/anchor-state-registry.md

```diff
@@ -0,0 +1,448 @@
+finality delay, and adding additional dependencies like blacklisting and game retirement. These concessions improve the
+confidence in resolved games, and calcify the assumptions upon which withdrawals and other dependents rest.
+
+## Definitions
+
+### Dispute game
+
+> See [Fault Dispute Game](fault-dispute-game.md)
+
+A dispute game is a contract that resolves an L2 state claim.
+
+### Maybe valid game
+
+A **maybe valid game** is a dispute game that correctly resolved in favor of the defender. However, the system concedes
+a possibility that it's not correct, and so it's not yet ready to be used as a **valid game** by dependents. A maybe
+valid game meets the following conditions:
+
+- Game was created by the dispute game factory.
+- Game is not **blacklisted**.
+- Game was created while it was the respected game type.
+- Game status is not `CHALLENGER_WINS`.
+- Game `createdAt` timestamp is less than the **game retirement timestamp**.
+
+### Finalized game
+
+A finalized dispute game is a game that has been resolved in favor of either the challenger or defender. Furthermore, it
+has passed the **dispute game finality delay** and can be used by dependents. A finalized game meets the following
+conditions:
+
+- Game status is `CHALLENGER_WINS` or `DEFENDER_WINS`.
+- Game `resolvedAt` timestamp is not zero.
+- Game `resolvedAt` timestamp is more than `dispute game finality delay` seconds ago.
+
+### Dispute game finality delay
+
+> Also known as "air gap."
+
+The dispute game finality delay is the period of time between a dispute game resolving and a dispute game becoming
+finalized. It's set via **authorized input**.
+
+### Valid game
+
+A game is a **valid game** if it, among other qualifications, has resolved in favor of the defender and has also matured
+past the finality delay. In other words, it meets the conditions of both a **maybe valid game** and a **finalized
+game**.
+
+### Blacklisted game
+
+A **blacklisted game** is a game that has been set as blacklisted via **authorized action**. It must not be considered
+valid, and must not be used for finalizing withdrawals or any other dependent L2-to-L1 action.
+
+### Invalid game
+
+An **invalid game** is a game whose claim was false, or does not meet some other **maybe valid game** condition.
+
+### Retired game
+
+A **retired game** is a game whose `createdAt` timestamp is older than the **game retirement timestamp**. A game that
+gets retired is no longer considered valid.
+
+### Game retirement timestamp
+
+The game retirement timestamp determines **retired games** and can only be adjusted via **authorized input**.
+
+### Anchor state
+
+> See [Fault Dispute Game -> Anchor State](fault-dispute-game.md#anchor-state).
+
+An anchor state is the state root from a correct L2 claim that is used as a starting point for new dispute games.
+
+### Anchor game
+
+The **anchor game** is the game whose state root is used as the **anchor state**. In the overwhelming majority of cases,
```

Comment:
A game can be marked as the "anchor game" if that game is currently a valid game and corresponds to a claim about an L2 block number greater than the L2 block number of the current anchor game. A game remains the anchor game until replaced by another anchor game. An anchor game that becomes blacklisted is no longer considered a usable anchor game. An anchor game that becomes retired is still considered a usable anchor game.